src/main/java/hudson/tasks/junit/HistoryTestResultSummary.java

```diff
@@ -11,24 +11,31 @@ public class HistoryTestResultSummary {
     private final int skipCount;
     private final int passCount;
     private final String description;
+    private final hudson.tasks.test.TestResult resultInRun;
```

Comment:
I think this would be clearer, can it be imported too?
```suggestion
    private final TestResult testResult;
```

Reply:
hudson.tasks.junit.TestResult != hudson.tasks.test.TestResult 
I don't remember the exact details anymore, but I think we are avoiding or introducing some lazy load on purpose here.

Import would be quite confusing because two classes with same name.